packages/app/src/systems/SignUp/components/MnemonicWrite/MnemonicWrite.test.tsx

```diff
@@ -1,21 +1,19 @@
-import { Mnemonic as FuelMnemonic } from "@fuel-ts/mnemonic";
-import { render, screen, waitFor } from "@fuel-ui/test-utils";
+import { Mnemonic as FuelMnemonic } from '@fuel-ts/mnemonic';
+import { render, screen, waitFor } from '@fuel-ui/test-utils';
 
-import { MnemonicWrite } from "./MnemonicWrite";
+import { MnemonicWrite } from './MnemonicWrite';
 
-import { MNEMONIC_SIZE } from "~/config";
-import { getPhraseFromValue } from "~/systems/Core";
+import { MNEMONIC_SIZE } from '~/config';
+import { getPhraseFromValue } from '~/systems/Core';
 
 const onFilledHandler = jest.fn();
 const onNextHandler = jest.fn();
 const onCancelHandler = jest.fn();
 
-const MNEMONIC = getPhraseFromValue(
-  FuelMnemonic.generate(MNEMONIC_SIZE)
-) as string;
+const MNEMONIC = getPhraseFromValue(FuelMnemonic.generate(MNEMONIC_SIZE)) as string;
```

Comment:
Also here the old rule for formatting was better, maybe the column number should be smaller